File: 백준/Bronze/2566. 최댓값/최댓값.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;


public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;
        int max = 0;
        int col = 0;
        int row = 0;
        for (int i = 0; i < 9; i++) {
            st = new StringTokenizer(br.readLine());
            for (int j = 0; j < 9; j++) {
                int num = Integer.parseInt(st.nextToken());
                if (max < num) {
                    max = num;
                    col = i + 1;
                    row = j + 1;
                } else if(max == 0){
                    col = 1;
                    row = 1;
                }
            }
        }
        System.out.println(max);
        System.out.println(col + " " + row);
    }
}

File: 백준/Bronze/2566. 최댓값/최댓값.cc
#include <iostream>
using namespace std;
const int COL = 9;
const int ROW = 9;
int arr[COL][ROW];
int main()
{
    int max = 0;
    int col, row;
    for (int i = 0; i < COL; i++)
    {
        for (int j = 0; j < ROW; j++)
        {
            cin >> arr[i][j];
        }
    }
    for (int i = 0; i < COL; i++)
    {
        for (int j = 0; j < ROW; j++)
        {
            if (arr[i][j] > max) {
                max = arr[i][j];
                col = i + 1;
                row = j + 1;
            } else if(max == 0)
            {
                col = 1;
                row = 1;
            }
        }
    }
    cout << max << '\n';
    cout << col << " " << row;
}